inc/abstracttarget.class.php

```diff
@@ -1501,6 +1512,79 @@ protected function showLocationSettings($rand) {
       echo '</tr>';
    }
 
+   protected function showValidationSettings($rand) {
+      echo '<tr>';
+
+      // Setting label
+      echo '<td width="15%">' . __('Validation') . '</td>';
+
+      // Possible values
+      echo '<td width="45%">';
+      Dropdown::showFromArray('validation_rule', static::getEnumValidationRule(), [
+         'value'     => $this->fields['validation_rule'],
+         'on_change' => "plugin_formcreator_change_validation($rand)",
+         'rand'      => $rand
+      ]);
+      echo Html::scriptBlock("plugin_formcreator_change_validation($rand)");
```

Comment:
I assume this JS function manipulates display attribut on subsequent DIV tags. To speedup clisnt side rendering, I think it would be better to set display attribute properly server side.